proto/stride/stakeibc/query.proto

```diff
@@ -147,3 +162,13 @@ message QueryAddressUnbondingsResponse {
   repeated AddressUnbonding address_unbondings = 1
       [ (gogoproto.nullable) = false ];
 }
+
+message QueryLSMDepositsRequest {
+  string chain_id = 1;
```

Comment:
why is chain id mandatory?

Reply:
I think I'd actually recommend just setting all these to strings. When someone wants to filter by status, it's an easier UX to pass the string than pass the status int.

And then there's no need to make these optional cause you can just check if the string is empty

Reply:
Yep as we discussed, avoid the optionals by making all strings and allow chain_id to be empty as well if you want to get all deposts.